teachertool/src/style.overrides/Input.scss

```diff
@@ -0,0 +1,50 @@
+.common-input-wrapper {
+
+    div[class*="common-input-group"] {
+        border-radius: 99px;
```

Comment:
I know this is just a border radius, but 99px seems like a high value, like we should be using something like rem with pixel values that high.

Reply:
This is a common hack to put circular borders on rectangular elements. The browser will adjust the actual border radius to match the element size.